app/code/Magento/ConfigurableProduct/Ui/DataProvider/Product/Form/Modifier/ConfigurablePanel.php

```diff
@@ -255,6 +256,29 @@ protected function getPanelChildren()
         return [
             'configurable_products_button_set' => $this->getButtonSet(),
             'configurable-matrix' => $this->getGrid(),
+            'show_product_listing' => $this->getProductListing(),
+        ];
+    }
+
+
+    /**
+     * Returns dynamic rows configuration
+     *
+     * @return array
+     */
+    protected function getProductListing()
```

Comment:
The new methods should be added with a `private` scope of visibility.

Reply:
fixed and committed